README.md

```diff
@@ -15,13 +15,13 @@ Please note: we strongly believe that **great companies always start with a frie
 Every post title **MUST** follow the format below:
 
 ```
-[Company Name] - [Job Title] - [Location]
+[Company Name] - [Job Title] - [Location] - [Salary]
```

Comment:
```suggestion
[Company Name] - [Job Title] - [Location] - [Salary] [Currency]/[Frequency]
```

Reply:
As per my comment in the main thread, I think showing they should add currency is a good idea.